src/plots/polar/polar.js

```diff
@@ -807,6 +788,23 @@ proto.updateMainDrag = function(fullLayout, polarLayout) {
         }
     };
 
+    dragOpts.clickFn = function(numClicks, evt) {
+        dragBox.removeZoombox(gd);
+
+        // TODO double once vs twice logic (autorange vs fixed range)
+        if(numClicks === 2) {
+            var updateObj = {};
+            for(var k in _this.viewInitial) {
+                updateObj[_this.id + '.' + k] = _this.viewInitial[k];
+            }
+
+            gd.emit('plotly_doubleclick', null);
+            Plotly.relayout(gd, updateObj);
+        }
+
+        Fx.click(gd, evt, _this.id);
```

Comment:
Hmm, this is not specific to polar, you're just replicating what we have elsewhere... but it occurs to me, having the `doubleClick` handled before `click` (in `Fx.click`) is both different from native events (which always seem to put the `doubleClick` last, though some browsers - [*cough* IE *cough*](https://stackoverflow.com/questions/5497073/how-to-differentiate-single-click-event-and-double-click-event) - don't emit the second `click` at all) AND potentially incorrect: if `Plotly.relayout` is synchronous, then `Fx.click` will make it look like you clicked on whatever point happens to be under your cursor after autoranging.

Reply:
There's https://github.com/plotly/plotly.js/issues/1546 that discuss the subject where @monfera argued for a new event called `plotly_singleclick` to go around this problem. Not sure what's best.

Reply:
Ah thanks for bringing up that issue - I'll just add this comment there.